Project/Src/Document/HighlightingStrategy/DefaultHighlightingStrategy.cs

```diff
@@ -65,6 +65,7 @@ static DefaultHighlightingStrategy()
                 ["CaretMarker"] = new HighlightColor(nameof(SystemColors.MenuBar), bold: false, italic: false),
                 ["CaretLine"] = new HighlightBackground(nameof(SystemColors.ControlLight), nameof(SystemColors.Window), bold: false, italic: false),
                 ["LineNumbers"] = new HighlightBackground(nameof(SystemColors.GrayText), nameof(SystemColors.Window), bold: false, italic: false),
+                ["LineNumberCurrent"] = new HighlightBackground(nameof(SystemColors.WindowText), nameof(SystemColors.Window), bold: true, italic: false),
```

Comment:
The base class used the term "SelectedLine" (and "SelectedFoldLine").
```suggestion
                ["SelectedLineNumber"] = new HighlightBackground(nameof(SystemColors.WindowText), nameof(SystemColors.Window), bold: true, italic: false),
```

Reply:
LineNumberSelected to keep closer to "LineNumbers"